challengeFive.js

```diff
@@ -0,0 +1,46 @@
+// First attempt 
```

Comment:
I can't run this file due to the way the two different functions are written. I have to comment out the second function to run the first function. When you push up code to a repository, it should be able to run without any issues.